packages/duppy/duppy.0.9.4/opam

```diff
@@ -0,0 +1,37 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Library providing monadic threads"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Romain Beauxis <toots@rastageeks.org>"]
+license: "LGPL-2.1-only"
+homepage: "https://github.com/savonet/ocaml-duppy"
+bug-reports: "https://github.com/savonet/ocaml-duppy/issues"
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {>= "2.7"}
+  "re"
```

Comment:
```suggestion
  "re" {>= "1.7.2"}
```
This is where `Re_pcre` (and similar) became `Re.Pcre`

Reply:
I see. Thanks!